pkg/minikube/cluster/cluster.go

```diff
@@ -282,17 +283,29 @@ func SetupCerts(d drivers.Driver) error {
 	return nil
 }
 
+func engineOptions(config MachineConfig) *engine.Options {
+
+	o := engine.Options{
+		Env: config.DockerEnv,
```

Comment:
Yeah, added a comment to the struct to clarify as well. libmachine handles the parsing.